addons/website_event/static/src/snippets/s_events/options.js

```diff
@@ -0,0 +1,55 @@
+/** @odoo-module **/
+
+import options from 'web_editor.snippets.options';
+import dynamicSnippetOptions from 'website.s_dynamic_snippet_options';
+
+const dynamicSnippetEventOptions = dynamicSnippetOptions.extend({
+    /**
+     * @override
+     */
+    init() {
+        this._super.apply(this, arguments);
+        this.modelNameFilter = 'event.event';
+        this.tagIDs = [];
+    },
+
+    async willStart() {
+        const _super = this._super.bind(this);
+        this.tagIDs = JSON.parse(this.$target[0].dataset.filterByTagIds || '[]');
+        const tags = await this._rpc({
+            model: 'event.tag',
+            method: 'search_read',
+            domain: ['&', '&', ['category_id.website_published', '=', true], ['color', '!=', false], ['color', '!=', 0]],
```

Comment:
Can't it be "color, not in, 0, false" ? I know sometimes this does not work as intended, but just asking if we can reduce domain length.